casper/src/main/scala/coop/rchain/casper/util/rholang/RhoTrieTraverser.scala

```diff
@@ -0,0 +1,182 @@
+package coop.rchain.casper.util.rholang
```

Comment:
Maybe to put these files in a separate folder because it's more specifically related to REV vault contract and not the Trie.

Reply:
which folder do you want to put?

Reply:
I am thinking putting everything in `coop/rchain/node/balance` now. What do you think?

Reply:
But I think the TreeHashMap traverse function should be useful for other developers.